### PR TITLE
Tracking links in Emails: Do not track hint

### PR DESCRIPTION
**Pull request by heebinho**
@escopecz pointed me to this hidden gem. I think this could be a worthy inclusion.

### CODE REVIEW

File: docs/channels/emails.rst [hunk 1]
```diff
@@ -263,7 +263,15 @@ Tracking links in Emails
 
 .. vale on
 
-Mautic tracks clicks of each link in an Email, with the stats displayed at the bottom of each Email detail view under the Click Counts tab.
+Mautic tracks clicks of each link in an Email, with the stats displayed at the bottom of each Email detail view under the Click Counts tab.  
+
+You can disable tracking for a certain link by adding the mautic:disable:tracking="true" html attribute.
```
Comment (escopecz) [inline docs/channels/emails.rst]:
```suggestion
You can disable tracking for a certain link by adding the `mautic:disable:tracking="true"` HTML attribute.  
```

**Review by escopecz — COMMENTED** [review]
Sweet! Thank you!

**Review by escopecz — DISMISSED** [review]

**Comment by RCheesley** [comment]
I think the changes I just committed should fix the Vale warnings. There were a couple of things - inline code blocks need a double backtick to encase them and I think you had too many spaces in the code block indentation - I just used VSCode and hit return on the end of the code block annotation to find out what it was expecting. Also changed 'disabled' to 'turn off' as we're trying to move towards more inclusive language.

Thanks for the PR @heebinho 🚀

**Review by RCheesley — APPROVED** [review]

**Comment by RCheesley** [comment]
@all-contributors please add @heebinho for docs

**Pull request merged**